Application/query.py

```diff
@@ -58,24 +58,13 @@ def getAllCategories(engine):
     return result
 
 def getAllOrders(engine):
-    orders = engine.execute("SELECT * FROM order")
+    orders = engine.execute("SELECT * FROM orders")
     result = []
 
     for row in orders:
         newRow = {"id": row[0], "customerId": row[1], 
                 "customerName": row[2], "date": row[3], 
-                "shipTo": row[4], "invoiceAmount": row[5]}
+                "shipTo": row[4], "invoiceAmount": int(row[5])}
```

Comment:
Since we are not building the main dashboard anymore, it's better to remove int()